glossary.tex

```diff
@@ -201,5 +201,56 @@
 \newglossaryentry{worker-node}
 {
   name={worker node},
-  description={See \gls{compute-node}}
+  description={See \gls{compute-node}},
+}
+\newglossaryentry{OpenStack}
+{
+  name={OpenStack},
+  description={OpenStack is a free and open-source software platform for cloud computing, mostly deployed as infrastructure-as-a-service (IaaS), whereby virtual servers and other resources are made available to customers},
+}
+\newglossaryentry{OpenStack Identity}
+{
+  name={OpenStack Identity},
+  description={OpenStack Identity (Keystone) provides a central directory of users mapped to the OpenStack services they can access. It acts as a common authentication system across the cloud operating system and can integrate with existing backend directory services. It supports multiple forms of authentication including standard username and password credentials, token-based systems and AWS-style logins.},
```

Comment:
use line breaks. it is not easy to give comment at the end of this very long line.

Reply:
Is this a general comment?  Should we add line breaks and re-indent the whole document?

Reply:
for th eline breaks, yes. max line length of 120 or so, but for text maybe 100? latex doesn't care and at least we can review it properly

wrt indent, i'm not sure what you mean here. i don't care about indentation unless latex requires it